testng-core/src/test/java/test/dataprovider/issue2819/SimpleRetry.java

```diff
@@ -3,19 +3,24 @@
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.UUID;
 import org.testng.IDataProviderMethod;
 import org.testng.IRetryDataProvider;
 
 public class SimpleRetry implements IRetryDataProvider {
```

Comment:
@juherr - I have altered this to start using a UUID instead of `hashCode()` because if we run with a static hashCode value then this test will always fail :) 

Here's one such instance https://github.com/cbeust/testng/actions/runs/3584269683/jobs/6030728277

Reply:
OK. Just rename things that refer to hashcode.

Reply:
Done